Makefile

```diff
@@ -61,12 +84,16 @@ deploy-targets:
 output-test: token-check
 	docker exec -it kind-control-plane crictl rmi $(PREFIX)/$(DOCKER_IMAGE):$(VERSION) || true
 	kind load docker-image $(PREFIX)/$(DOCKER_IMAGE):$(VERSION) --name kind
+
+	docker exec -it kind-control-plane crictl rmi $(PREFIX)/test-proxy:$(VERSION) || true
+	kind load docker-image $(PREFIX)/test-proxy:$(VERSION) --name kind
+
 	(cd $(KUSTOMIZE_DIR) && ./test.sh nimba $(WAVEFRONT_API_KEY) $(VERSION))
 
 token-check:
 	if [ -z ${WAVEFRONT_API_KEY} ]; then echo "Need to set WAVEFRONT_API_KEY" && exit 1; fi
 
-full-loop: token-check build tests container output-test
+full-loop: token-check build tests containers output-test
```

Comment:
Should we build the text proxy container as part of the full-loop? Do we expect it to change regularly?

Reply:
We looked at this and there are a few open questions around this, so we put some items on the technical architecture retro backlog